apps/react-18-tests-v9/package.json

```diff
@@ -19,14 +19,17 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@cypress/react": "9.0.1",
```

Comment:
I guess this is wrong ? <img width="1976" alt="image" src="https://github.com/user-attachments/assets/a88dc0af-0d43-450a-ab1d-9ec278dd7004" />